packages/flutter/lib/src/material/text_field.dart

```diff
@@ -158,7 +159,7 @@ class TextField extends StatefulWidget {
   /// field.
   final ValueChanged<String> onSubmitted;
 
-  /// Optional input validation and formatting overrides. Formatters are run 
+  /// Optional input validation and formatting overrides. Formatters are run
```

Comment:
You didn't introduce this, but since you touched this line, can you move the second sentence into its own paragraph?